### ISSUE TITLE
Level Design?

### ISSUE THREAD

**Issue by mcpatrickf5** [danielkennethgoto/cpts122#5]
Personally I think we should meet and go over this in person, but if anyone has something for level design in mind that works too. One thing is we should decide on how we want to end the level (eg door, computer terminal, display "Victory"-esque screen, etc).

**Comment by ReidReininger** [comment]
At the moment I'm thinking we just have an object that you touch and it activates an end of level cut scene. Also, as far as designing where the obstacles go, is anyone else familiar with gimp?

**Comment by mcpatrickf5** [comment]
I have gimp but never learned how to use it. Can it tell you pixel positions and whatnot?

**Comment by ReidReininger** [comment]
Yes and there's a clipboard that we can add images of our objects to and just click and drag onto the background. It's really easy to move things around.
